tutorial/bluetooth/README.md

```diff
@@ -0,0 +1,48 @@
+# Bluetooth Tutorial
+
+## What you need
+
+    - Pimoroni Badger2040W
+    - Personal computer with Go 1.22 and TinyGo 0.32 installed, and a serial port.
+
+## Code
+
+### step1.go - Bluetooth scan
+
+Run the code.
+
+```shell
+tinygo flash -target badger2040-w -stack-size=8kb -monitor ./tutorial/bluetooth/step1
+```
+
+You will see output on your terminal.
+
+### step2.go - Bluetooth scan on Badger2040-W display
+
+Run the code.
+
+```shell
+tinygo flash -target badger2040-w -stack-size=8kb ./tutorial/bluetooth/step2
```

Comment:
while not really needed, for consistency I'll add the -monitor to the command too, what do you think?